cineast-core/src/main/java/org/vitrivr/cineast/core/features/abstracts/AbstractTextRetriever.java

```diff
@@ -80,11 +80,6 @@ public void init(PersistencyWriterSupplier phandlerSupply, int batchSize) {
     this.writer = new SimpleFulltextFeatureDescriptorWriter(phandlerSupply.get(), this.tableName, batchSize);
   }
 
-  @Override
-  public void processSegment(SegmentContainer shot) {
```

Comment:
I don't see why you would remove this and then re-add it twice in implementing classes, especially since there are no plans to add this for Description, TagsFt and AudioTranscription

Reply:
You are right, I will undo this in the upcoming commit.